app/javascript/components/Admin/SearchPopover.tsx

```diff
@@ -161,7 +161,7 @@ const SearchPopover = () => {
 
         <form onSubmit={(e) => submitForm(e, Routes.admin_cards_path(), null)} style={{ display: "contents" }}>
           <select name="card_type" value={data.card_type} onChange={(e) => setData("card_type", e.target.value)}>
-            <option>Choose card type</option>
+            <option value="">Choose card type</option>
```

Comment:
This will make the option act as a placeholder, resolving the bug in PR description